modules/cryptocoin.py

```diff
@@ -121,9 +138,11 @@ def prettify(thing):
 @commandHook(['coin'])
 def coin(irc, ev):
     try:
-        coin = ev.args[0]
+        coin = coinmap[ev.args[0].lower()]
```

Comment:
Isn't it easier if you use `coinmap.get(ev.args[0].lower(), even.args[0])`? You won't need the extra except block that way.